package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "lab-03-fs-async",
+  "version": "1.0.0",
+  "description": "lab-03",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha, chai"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ronoh2001/lab-03-fs-async.git"
+  },
+  "author": "ronoh2001@gmail.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ronoh2001/lab-03-fs-async/issues"
+  },
+  "homepage": "https://github.com/ronoh2001/lab-03-fs-async#readme",
+  "dependencies": {
```

Comment:
gulp should be in your devDependencies because your code does not require it to run.